ioctl/cmd/ws/wsprojectconfig.go

```diff
@@ -113,17 +131,21 @@ func generateProjectFile(version, vmType, codeFile, confFile, expParam, outputFi
 	}
 
 	var (
-		confMaps  = make([]map[string]interface{}, 0)
 		confMap   = make(map[string]interface{})
+		verMap    = make(map[string]interface{})
+		verMaps   = make([]map[string]interface{}, 0)
 		outputMap = make(map[string]interface{})
 	)
 
 	if expParam != "" {
-		confMap["codeExpParam"] = expParam
+		verMap["codeExpParam"] = expParam
+	}
+	verMap["vmType"] = string(tye)
+	verMap["code"] = hexString
```

Comment:
maybe we can move code to lowest line, to make project file more readable

Reply:
add `projectConfig` struct for readable